sympy/assumptions/facts.py

```diff
@@ -34,71 +36,138 @@ def get_composite_predicates():
 
 @cacheit
 def get_known_facts():
+    """
+    Facts between unary predicates.
+
+    Returns
+    =======
+
+    x : Placeholder symbol for unary facts.
+
+    fact : Known facts in conjugated normal form.
+
+    """
     # We build the facts starting with primitive predicates.
     # DO NOT include the predicates in get_composite_predicates()'s keys here!
-    return And(
-
+    x = Symbol('x')
+    fact = And(
         # primitive predicates exclude each other
-        Implies(Q.negative_infinite, ~Q.positive_infinite),
-        Implies(Q.negative, ~Q.zero & ~Q.positive),
-        Implies(Q.positive, ~Q.zero),
+        Implies(Q.negative_infinite(x), ~Q.positive_infinite(x)),
+        Implies(Q.negative(x), ~Q.zero(x) & ~Q.positive(x)),
+        Implies(Q.positive(x), ~Q.zero(x)),
 
         # build real line and complex plane
-        Implies(Q.negative | Q.zero | Q.positive, ~Q.imaginary),
-        Implies(Q.negative | Q.zero | Q.positive | Q.imaginary, Q.algebraic | Q.transcendental),
+        Implies(Q.negative(x) | Q.zero(x) | Q.positive(x), ~Q.imaginary(x)),
+        Implies(Q.negative(x) | Q.zero(x) | Q.positive(x) | Q.imaginary(x),
+                Q.algebraic(x) | Q.transcendental(x)),
```

Comment:
This should just be
```python
Q.complex(x) >> (Q.algebraic(x) | Q.transcendental(x))
```

Reply:
In fact this is an equivalence rather than an implication

Reply:
The reverse implication is already known

```py
>>> satask(Q.complex(x) << (Q.algebraic(x) | Q.transcendental(x)))
True
>>> satask(Q.algebraic(x) | Q.transcendental(x), Q.complex(x))
True
```

(although browsing through the facts it isn't clear to me exactly why)